CMakeLists.txt

```diff
@@ -118,7 +118,15 @@ else()
   endif()
 endif()
 
+get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
```

Comment:
This is horribly wrong. It's a very bad idea to rely on some random CMake internals to guess this, and even if it solves anything, it solves only 1 out of 5 or 6 different scenarios.